pkg/apis/core/v1alpha1/types_seed.go

```diff
@@ -193,6 +197,14 @@ type SeedSettingScheduling struct {
 	Visible bool `json:"visible" protobuf:"bytes,1,opt,name=visible"`
 }
 
+// SeedSettingLoadBalancerServices controls certain settings for services of type load balancer that are created in the
+// seed.
+type SeedSettingLoadBalancerServices struct {
+	// Annotations is a map of annotations that will be injected/merged into every load balancer service object.
+	// +optional
+	Annotations map[string]string `json:"annotations,omitempty" protobuf:"bytes,1,opt,name=annotations"`
```

Comment:
```suggestion
	Annotations map[string]string `json:"annotations,omitempty" protobuf:"bytes,1,rep,name=annotations"`
```
To be consistent with https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/types.go#L238.

Reply:
Thanks, I'm wondering why this stuff was not automatically generated correctly...